### PR TITLE
arcconf controller specific tests - v2

### PR DESCRIPTION
**Pull request by manvanthar**
arcconf_cntl_oper.py tests arcconf tools controller specific configs.

addressed all the suggestions in v1.

Signed-off-by: Manvanthara B Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by manvanthar** [comment]
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/625308/job.txt)
[results.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/625309/results.txt)

**Comment by narasimhan-v** [comment]
Looks good to me

File: io/disk/arcconf/arcconf_cntl_oper.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        if not smm.check_installed("arcconf"):
+            if detected_distro.name == "Ubuntu":
+                http_repo = "%s%s.deb" % (self.http_path, self.tool_name)
+                self.repo = self.fetch_asset(http_repo, expire='10d')
+                cmd = "dpkg -i %s" % self.repo
+            else:
+                http_repo = "%s%s.rpm" % (self.http_path, self.tool_name)
+                self.repo = self.fetch_asset(http_repo, expire='10d')
+                cmd = "rpm -ivh %s" % self.repo
+            if process.system(cmd, shell=True) == 0:
```
Comment (PraveenPenguin) [inline io/disk/arcconf/arcconf_cntl_oper.py]: just one change please use `ignore_status=True` when it fails script will error out

**Comment by manvanthar** [comment]
addressed in v3

**Pull request closed without merging**